biodatasets/chemdner/chemdner.py

```diff
@@ -0,0 +1,392 @@
+year={2015},
+month={Jan},
+day={19},
+volume={7},
+number={1},
+pages={S2},
+abstract={
+    The automatic extraction of chemical information from text requires the recognition of chemical entity mentions
+    as one of its key steps. When developing supervised named entity recognition (NER) systems, the availability of
+    a large, manually annotated text corpus is desirable. Furthermore, large corpora permit the robust evaluation
+    and comparison of different approaches that detect chemicals in documents. We present the CHEMDNER corpus,
+    a collection of 10,000 PubMed abstracts that contain a total of 84,355 chemical entity mentions labeled manually
+    by expert chemistry literature curators, following annotation guidelines specifically defined for this task.
+    The abstracts of the CHEMDNER corpus were selected to be representative for all major chemical disciplines.
+    Each of the chemical entity mentions was manually labeled according to its structure-associated chemical entity
+    mention (SACEM) class: abbreviation, family, formula, identifier, multiple, systematic and trivial.
+    The difficulty and consistency of tagging chemicals in text was measured using an agreement study between
+    annotators, obtaining a percentage agreement of 91. For a subset of the CHEMDNER corpus
+    (the test set of 3,000 abstracts) we provide not only the Gold Standard manual annotations, but also
+    mentions automatically detected by the 26 teams that participated in the BioCreative IV CHEMDNER
+    chemical mention recognition task. In addition, we release the CHEMDNER silver standard corpus of automatically
+    extracted mentions from 17,000 randomly selected PubMed abstracts. A version of the CHEMDNER corpus in
+    the BioC format has been generated as well. We propose a standard for required minimum information about
+    entity annotations for the construction of domain specific corpora on chemical and drug entities.
+    The CHEMDNER corpus and annotation guidelines are available at:
+    http://www.biocreative.org/resources/biocreative-iv/chemdner-corpus/
+},
+issn={1758-2946},
+doi={10.1186/1758-2946-7-S1-S2},
+url={https://doi.org/10.1186/1758-2946-7-S1-S2}
+}
+"""
+
+_DESCRIPTION = """\
+We present the CHEMDNER corpus, a collection of 10,000 PubMed abstracts that contain a total of 84,355 chemical entity
+mentions labeled manually by expert chemistry literature curators, following annotation guidelines specifically
+defined for this task. The abstracts of the CHEMDNER corpus were selected to be representative for all major chemical
+disciplines. Each of the chemical entity mentions was manually labeled according to its structure-associated chemical
+entity mention (SACEM) class: abbreviation, family, formula, identifier, multiple, systematic and trivial.
+"""
+
+_HOMEPAGE = "https://biocreative.bioinformatics.udel.edu/resources/biocreative-iv/chemdner-corpus/"
+_LICENSE = "Creative Commons Attribution 4.0 International"

```

Comment:
I think we still need to update this (https://github.com/bigscience-workshop/biomedical/pull/292#discussion_r840205588)

Reply:
No response from the main author for now.

Reply:
Let's change it to "Unknown" for now.

Reply:
Ok